nemo/lightning/io/api.py

```diff
@@ -164,7 +171,8 @@ def import_ckpt(
 
     importer: ModelConnector = model.importer(source)
     ckpt_path = importer(overwrite=overwrite, output_path=output_path)
-    importer.on_import_ckpt(model)
+    if import_tokenizer:
+        importer.on_import_ckpt(model)
```

Comment:
can you make import_tokenizer a arg to `on_import_ckpt` as well? `on_import_ckpt` is supposed to be a general hook, you can change inside `on_import_ckpt` instead of here.